packages/rapid-ui/src/components/primitives/box/Box.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { RapidStyles } from "../../../utils";
+
+type BoxType = "div";
+
+interface BoxProps extends React.HTMLAttributes<HTMLDivElement> {
+  styles?: string;
+  as?: BoxType;
+}
+
+const THEME_CLASSNAME = "rapid-box";
+
+const Box = React.forwardRef<HTMLDivElement, BoxProps>(
+  ({ styles, as = "div", ...rest }, ref) => {
+    const Component = (type: BoxType) => {
+      switch (type) {
+        case "div":
+          return (
+            <div
+              ref={ref}
+              className={RapidStyles(
+                styles || rest.className,
```

Comment:
We need a third param passed into here in-between the className and THEME_CLASSNAME - just use an empty string for the default styes ""